app/utils/draw.py

```diff
@@ -90,14 +90,18 @@ def draw_graph_with_weights(canvas, graph, center_indices=None):
             neighbour_x = center[0] + r * sin(neighbour_angle)
             neighbour_y = center[1] - r * cos(neighbour_angle)
 
-            canvas.create_line(x, y, neighbour_x, neighbour_y)
+            if minimal_tree and neighbour in minimal_tree.graph[i]:
+                canvas.create_line(x, y, neighbour_x, neighbour_y, fill=color, width=2)
```

Comment:
it's outside of class, should use some method instead of access to "private" field